
uint8 where
---
uint8[] lista
uint8 AZUL=13
uint8 VERDE=12
uint8 VERMELHO=14
uint8 DESCONHECIDO=255